org.lflang/src/org/lflang/federated/FedCLauncher.java

```diff
@@ -58,16 +60,34 @@ public FedCLauncher(
      * Return the compile command for a federate.
      * 
      * @param federate The federate to compile.
+     * @throws IOException 
      */
     @Override
     protected
     String compileCommandForFederate(FederateInstance federate) {
-        CCompiler cCompiler = new CCompiler(targetConfig, fileConfig, errorReporter);
-        return String.join(" ", 
+        FedFileConfig fedFileConfig = null;
+        TargetConfig localTargetConfig = targetConfig;
+        try {
+            fedFileConfig = new FedFileConfig(fileConfig, federate.name);
+        } catch (IOException e) {
+            errorReporter.reportError("Failed to create file config for federate "+federate.name);
+            return "";
+        }
+        
+        String commandToReturn = "";
+        // FIXME: Hack to add platform support only for linux systems. 
```

Comment:
This seems worth creating an issue for.

Reply:
Added a comment to the relevant outstanding issue #329.